packages/cli/globalSetup.js

```diff
@@ -52,7 +52,7 @@ export default async function globalSetup() {
 
   await setup({
     command:
-      `IPFS_PATH=\'${GOIPFS_DIR_PATH}\' CERAMIC_ENABLE_EXPERIMENTAL_INDEXING=\'true\' rm -rf ~/.goipfs && pnpm dlx @ceramicnetwork/cli daemon --config ${CONFIG_PATH}`,
+      `IPFS_PATH=\'${GOIPFS_DIR_PATH.pathname}\' CERAMIC_ENABLE_EXPERIMENTAL_INDEXING=\'true\' pnpm dlx @ceramicnetwork/cli daemon --config ${CONFIG_PATH.pathname}`,
```

Comment:
Looks like `CERAMIC_ENABLE_EXPERIMENTAL_INDEXING` is set both here and in `jest-environment-composedb`, could it be removed from whichever place it's not needed please?

Reply:
Another thing I noticed running tests locally, looks like the Ceramic node connects to Clay, but that shouldn't be needed, is it? Can't we use the `inmemory` network to avoid having CI rely on network connectivity?

Reply:
> Looks like CERAMIC_ENABLE_EXPERIMENTAL_INDEXING is set both here and in jest-environment-composedb, could it be removed from whichever place it's not needed please?

@PaulLeCam , `cli` test don't use `jest-environment-composedb` at this time and they use the `globalSetup.js` instead. I'm not sure, if it's the right time now to refactor it, WDYT?

Reply:
Also, just for clarity, `globalSetup.js` is currently only used in `cli`.